examples/quickstarts/archetypes/set-version.sh

```diff
@@ -75,4 +75,14 @@ for _ex in ${EXAMPLES}; do
 
 done
 
+# Update helidon.version property in bare archetypes
```

Comment:
This should be solved in a pure Maven way.
See the following diff:

```dif
diff --git a/archetypes/mp/src/main/resources/archetype-resources/pom.xml b/archetypes/mp/src/main/resources/archetype-resources/pom.xml
index 167a8d9a..711406cd 100644
--- a/archetypes/mp/src/main/resources/archetype-resources/pom.xml
+++ b/archetypes/mp/src/main/resources/archetype-resources/pom.xml
@@ -11,7 +11,7 @@
     <name>${artifactId}</name>
 
     <properties>
-        <helidon.version>1.2.2-SNAPSHOT</helidon.version>
+        <helidon.version>@project.version@</helidon.version>
 #if( $mainClass.matches("y|yes|true") )
         <mainClass>${package}.Main</mainClass>
 #else
diff --git a/archetypes/pom.xml b/archetypes/pom.xml
index 93c27566..c218caef 100644
--- a/archetypes/pom.xml
+++ b/archetypes/pom.xml
@@ -34,4 +34,41 @@
         <module>mp</module>
         <module>se</module>
     </modules>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>archetype-resources/pom.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <excludes>
+                    <exclude>archetype-resources/pom.xml</exclude>
+                </excludes>
+            </resource>
+        </resources>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-resources-plugin</artifactId>
+                    <executions>
+                        <execution>
+                            <id>default-resources</id>
+                            <configuration>
+                                <delimiters>
+                                    <delimiter>@</delimiter>
+                                </delimiters>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+    </build>
 </project>
diff --git a/archetypes/se/src/main/resources/archetype-resources/pom.xml b/archetypes/se/src/main/resources/archetype-resources/pom.xml
index ec95b917..37c69244 100644
--- a/archetypes/se/src/main/resources/archetype-resources/pom.xml
+++ b/archetypes/se/src/main/resources/archetype-resources/pom.xml
@@ -11,7 +11,7 @@
     <name>${artifactId}</name>
 
     <properties>
-        <helidon.version>1.2.2-SNAPSHOT</helidon.version>
+        <helidon.version>@project.version@</helidon.version>
 #if( $nativeImageSupport.matches("y|yes|true") )
         <helidon.plugin.version>1.0.10</helidon.plugin.version>
 #end
```

Reply:
I've tossed the initial commit and replaced it with the above.